packages/core/upload/admin/src/components/TableList/index.js

```diff
@@ -2,9 +2,13 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { useIntl } from 'react-intl';
 import { BaseCheckbox } from '@strapi/design-system/BaseCheckbox';
+import { IconButton } from '@strapi/design-system/IconButton';
 import { Table, Th, Thead, Tr } from '@strapi/design-system/Table';
+import { Tooltip } from '@strapi/design-system/Tooltip';
 import { Typography } from '@strapi/design-system/Typography';
 import { VisuallyHidden } from '@strapi/design-system/VisuallyHidden';
+import CarretDown from '@strapi/icons/CarretDown';
```

Comment:
you can just do 
```suggestion
import { CarretDown, CarretUp } from '@strapi/icons';
```
because now we have a better built DS.